com.jetpackduba.Gitnuro.metainfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>com.jetpackduba.Gitnuro</id>
```

Comment:
`http://jetpackduba.com/` redirect to an unconfigured website. is this app-id really valid?

Reply:
Yeah, I've just bought the domain this week, I'm currently working on the website.

Reply:
gitnuro.jetpackduba.com is already up! (Still waiting for Gitlab pages to setup the SSL cert tho)